src/libs/actions/IOU.ts

```diff
@@ -5338,11 +5339,12 @@ function requestMoney(requestMoneyInformation: RequestMoneyInformation) {
         }
         default: {
             // This is only required when inviting admins to test drive the app
+            const {onboardingMessages} = getOnboardingMessages();
             const guidedSetupData: GuidedSetupData | undefined = isTestDrive
                 ? prepareOnboardingOnyxData(
                       {choice: CONST.ONBOARDING_CHOICES.TEST_DRIVE_RECEIVER},
                       CONST.ONBOARDING_CHOICES.TEST_DRIVE_RECEIVER,
-                      CONST.ONBOARDING_MESSAGES[CONST.ONBOARDING_CHOICES.TEST_DRIVE_RECEIVER],
+                      onboardingMessages[CONST.ONBOARDING_CHOICES.TEST_DRIVE_RECEIVER],
```

Comment:
```suggestion
                      getOnboardingMessages().onboardingMessages[CONST.ONBOARDING_CHOICES.TEST_DRIVE_RECEIVER],
```
NAB. Just trying to avoid calling the function if we are not going to use its result